src/server.h

```diff
@@ -1199,9 +1114,73 @@ typedef struct {
 } clientReqResInfo;
 #endif
 
+#define CLIENT_BITFIELD_FLAGS                                                                                          \
```

Comment:
We dropped all of the documentation around the flags, it would be nice if we could keep them in the header file in some capacity. In one of my other comments I suggested we drop the anonymous struct variant, then we can just document the variables in the cflags struct.